docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '2'
+services:
+  libs-builder:
+    build : .
+    image: malos-vision-android-libs
+    command: 
+      - /bin/sh
+      - -c 
+      - |
+        cd /opt/src/
+        rm -rf caffe-android-lib && rm -rf android_vision_libs*
+        git clone https://github.com/hpsaturn/caffe-android-lib.git && \
+        cd caffe-android-lib && git checkout av/maloslib && git clone https://github.com/matrix-io/matrix-malos-lib.git && git clone https://github.com/matrix-io/protocol-buffers.git matrixio_protos && \
```

Comment:
please change it to master, caffe-androd-lib already have merged maloslib branch

Reply:
great , i changed here, just testing right now